index.js

```diff
@@ -30,6 +30,9 @@ module.exports = function gulpPug(options) {
           log('compiling file', file.path);
         }
         if (opts.client) {
+          if (typeof options.name === 'function') {
```

Comment:
`options` may be undefined, use `opts.name`.

Reply:
If I use `opts.name` I will need the temp variable, you rejected in another comment. Otherwise it won't work for multiple files.